src/pages/components/modal/usage.mdx

```diff
@@ -15,6 +15,15 @@ information via a window that sits on top of the page content.
 
 </PageDescription>
 
+<InlineNotification>
+
+Experimental Slug is now available for
```

Comment:
@annawen1 are we keeping it in sentence case only? i.e. "Experimental slug"?

Reply:
Oh yes, good catch. No capitalizing the component names.

Reply:
Thanks @aagonzales!